napari/utils/events/dataclass.py

```diff
@@ -223,9 +251,15 @@ def evented_post_init(self: T, *initvars) -> None:
             for em in self.events.emitters:
                 e_fields.pop(em, None)
             self.events.add(**e_fields)
+            if "values_updated" not in self.events.emitters:
+                self.events.add(values_updated=None)
         else:
             self.events = EmitterGroup(
-                source=self, auto_connect=False, **e_fields
+                source=self,
+                auto_connect=False,
+                deprecated=getattr(cls, "deprecated_events", None),
+                values_updated=None,
```

Comment:
What is the reason for the `values_updated` event?

Reply:
It looks like the individual events will get emitted when they are set. Is this just an extra event to let you know the whole dataclass is updated?

Reply:
I'm so so on the addition of that event as well.  I'd tend towards waiting to find a use case where just connecting directly to specific events is insufficient before we start adding events that might be useful someday.

Reply:
For example camera synchronization. 
Maybe refactor VispyCamera which could be a little simplified. 

For me, it covers all cases when we would like to use `as_dict` Then we do not need to know (on the level of implementation) exactly the list of parameters (signals). 

Similar need we could observe in many places where there is near the same code to change different parameters of the object.

Reply:
The normal events keep on getting emitted to though, so you have really have to decide now do you listen to individual events or to the `update` event as doing both will cause to double responses.

Also should we then check the dict for equality to see if any properties change and event actually needs to be emitted (if we still do that check for individual properties?).

You can also connect to `dataclass.events` if you want an event each time any property is changed, which is different, but now we have three different ways to listen to events which I think is too much.

Like @tlambert03 said, we could add this at some point if we found that we wanted to start using it internally (and camera might be the right place to see), but I'm hesitant to expand the API right now unless we absolutely must

Reply:
My bad. I missed the connection to the emitter group.

Reply:
removed